src/components/Header/components/SearchButton.tsx

```diff
@@ -15,28 +15,36 @@ const SearchButton = chakra(({ isMobile = false }: SearchButtonProps) => {
   const styles = useStyleConfig("HeaderSearchButton", { isMobile, isOpen });
   const buttonText = isMobile ? null : isOpen ? "Close" : "Search";
   const labelText = isOpen ? "Close Search" : "Open Search";
+  const ref = useRef<HTMLDivElement>(null);
+
+  useOutsideClick({
```

Comment:
YEah, same issue as above. It works within the header component but not outside of it and it could be because of Storybook. Will further investigate in a future PR.